example/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "TurboWebviewExample",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "pods": "pod-install --quiet",
+    "server": "yarn --cwd server"
+  },
+  "dependencies": {
+    "react": "18.1.0",
+    "react-native": "0.70.3",
+    "@react-navigation/bottom-tabs": "^6.3.1",
+    "@react-navigation/native": "^6.0.10",
+    "@react-navigation/native-stack": "^6.6.2",
+    "react-native-safe-area-context": "^4.3.1",
+    "react-native-screens": "^3.13.1"
+  },
```

Comment:
Assuming in the future this will have the deps to `react-native-turbo`?

Reply:
Right now any changes to the root folder library js/ts code will be reflected without rebuild. Thats why its configured with 
```js
    'react-native-turbo-webview': {
      root: path.join(__dirname, '..'),
    },
 ```
https://github.com/software-mansion-labs/react-native-turbo-demo/blob/08bfb3606d5937c9296090e6bbe7a3d265928b0e/example/react-native.config.js not with the package.json. Please let me know if I have misunderstood the question.